week2/index.js

```diff
@@ -0,0 +1,41 @@
+import fs from 'fs';
+const argumentArray = process.argv;
+const command = process.argv[2];
+const argument = process.argv[3];
+switch (command) {
+  case "help":
+  case undefined:
+    fs.readFile('help.txt', 'utf8', (err, data) => {
+      if (err) throw err;
+      console.log(data);
+    });
+    break;
+  case "add":
+    fs.appendFile('todoList.txt', "\n" + argument, (err) => {
```

Comment:
The problem with appending `\n` before `argument` is that the result of this command `node index.js add "task 5"` would be 

```
task 0
task 1
task 2
task 3
task 4

task 5
```

the first time we use the app. So we don't really want to add a newline character before, but after the `argument`. To make sure that it is always there!